src/components/Badge/Badge.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { describe, expect, test } from 'vitest';
+import { screen } from '@testing-library/react';
+import { Badge } from '.';
+import contextRender from '../../shared/utils/contextRender';
+
+describe('Badge component', () => {
+  test('Badge should render correctly', () => {
+    contextRender(
+      <Badge severity={'error'} isStrong={false}>
+        Error
+      </Badge>
+    );
+    const badge = screen.getByText('Error');
+    expect(badge).toBeInTheDocument();
+  });
+
+  test('Badge isStrong should change', () => {
+    contextRender(
+      <Badge severity={'error'} isStrong={true}>
+        Error
+      </Badge>
+    );
+    const badge = screen.getByText('Error');
+    expect(badge).toBeInTheDocument();
+    screen.debug(badge);
```

Comment:
Could you please remove this test, since we didn't get it working?

For anyone else following along, I asked Clae to add that `screen.debug()` line to see what was in the DOM that could be used for writing unit tests against. But there wasn't a whole lot. 

Looks like emotion recommends snapshot testing, see https://emotion.sh/docs/testing ,  but it needs an extra package to do that.

Reply:
I am not sure if I would do snapshot testing because I find it difficult to maintain and would always find snapshot failures here and there. But I do encourage to give it a try and explore.

Reply:
everything looks good - going to approve